views/agronomical-object/_form.php

```diff
@@ -0,0 +1,239 @@
+                    type: 'text',
+                    required: false,
+                    readOnly: true
+                }
+
+            ],
+            rowHeaders: true,
+            colHeaders: [
+                "<b><?= Yii::t('app', 'Generated URI') ?></b>",
+                "<b><?= Yii::t('app', 'Alias') ?></b>",
+                "<b><?= Yii::t('app', 'Type') ?></b>",
+                "<b><?= Yii::t('app', 'Experiment') ?></b>",
+                "<b><?= Yii::t('app', 'Geometry') ?></b>",
+                "<b><?= Yii::t('app', 'Parent') ?></b>",
+                "<b><?= Yii::t('app', 'Species') ?></b>",
+                "<b><?= Yii::t('app', 'Variety') ?></b>",
+                "<b><?= Yii::t('app', 'Insertion status') ?></b>"
+            ],
+            manualRowMove: true,
+            manualColumnMove: true,
+            contextMenu: true,
+            filters: true,
+            dropdownMenu: true,
+            cells: function(row, col, prop) {
+                var cellProperties = {};
+                
+                if (col === 0) {
+                    cellProperties.renderer = uriColumnRenderer;
+                }
+                
+                return cellProperties;
+            },
+            afterGetColHeader: function (col, th) {
+                if (col === 1 | col === 2 | col === 3 ) {
+                    th.style.color = "red";
+                }
+            }
+            
+         });
+         
+        /**
+         * if the data is valid, calls the insert action
+         * @param {boolean} callback
+         * @returns 
+         */
+        function add(callback) {
+            if (callback) {
+                document.getElementById("loader").style.display = "block";
+                document.getElementById("objects-creation").style.display = "none";
+                
+                var objectsArray = handsontable.getData();
+                var objectsString = JSON.stringify(objectsArray);
+                $.ajax({
+                    url: 'index.php?r=agronomical-object%2Fcreate-multiple-scientific-objects',
+                    type: 'POST',
+                    dataType: 'json',
+                    data: {objects: objectsString}
+                }).done(function (data) {
+                    document.getElementById("objects-creation").style.display = "block";
+                    document.getElementById("loader").style.display = "none";
+                    for (var i = 0; i < data["messages"].length; i++) {
+                        if (data["objectUris"][i] !== null) {
+                            handsontable.setDataAtCell(i, 0, data["objectUris"][i]);
+                        }
+                        handsontable.setDataAtCell(i, 8, data["messages"][i]);
+                    }                    
+                    $('#objects-save').hide();
+                })
+                .fail(function (jqXHR, textStatus) {
+                    console.log(jqXHR.responseText);
```

Comment:
Remove console.log and instead display an error message with a toaster like that in JS:
// Toastr options generated by @see https://codeseven.github.io/toastr/demo.html
toastr.options = {
    "closeButton": false,
    "debug": false,
    "newestOnTop": false,
    "progressBar": false,
    "positionClass": "toast-top-right",
    "preventDuplicates": false,
    "onclick": null,
    "showDuration": "300",
    "hideDuration": "1000",
    "timeOut": "2000",
    "extendedTimeOut": "1000",
    "showEasing": "swing",
    "hideEasing": "linear",
    "showMethod": "fadeIn",
    "hideMethod": "fadeOut"
};

toastr\["error"\](message);